weapons/forgotten-weapons.yaml

```diff
@@ -630,7 +628,7 @@ GeckoLauncherBarrage:
 ForgottenT3Missile:
 	Inherits: ^MissileAPHE
 	Inherits@wh: ^APHEWH
-	ValidTargets: Ground, Air, MidAir
+	ValidTargets: Ground, Aircraft
```

Comment:
Should get split to AG and AA version to avoid warhead attacking air and ground simultaneously